DataContent/ReadingDB/ComputerDataService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using DataManipulation.DataFillers;
+using ItemLibrary;
+using ItemLibrary.DataContexts;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace DataContent.ReadingDB.Services
+{
+    public class ComputerDataService :IData<IEnumerable<Computer>>
+    {
+        private ComputerContext _db { get; set; }
```

Comment:
here we could implement Dependency Injection i guess.  we should set DBContext via constructor and then use it. it would be something like: 

private DbContext _db;

public ComputerDataService (DbContext db)
{
    _db = db;
}

Reply:
I tried, however it seemed that then it doesn't see sets like Computers